docs/source/examples/validation_example.ipynb

```diff
@@ -0,0 +1,392 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Validation of EIS data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### The Kramers-Kronig Relations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Electrochemical impedance spectroscopy (EIS) is built on linear systems theory which requires that the system satisfy conditions of causality, linearity, and stability. The Kramers-Kronig relations consist of a set of transformations that can be used to predict one component of the impedance from the other over the frequency limits from zero to infinity. For example, one might calculate the imaginary component of the impedance from the measured real component, \n",
+    "$$\n",
+    "Z^{\\prime\\prime}(\\omega) = - \\frac{2\\omega}{\\pi} \\int_0^\\infty \\frac{Z^{\\prime}(x) - Z^{\\prime}(\\omega)}{x^2 - \\omega^2}dx\n",
+    "$$\n",
+    "where $Z^{\\prime}(\\omega)$ and $Z^{\\prime\\prime}(\\omega)$ are the real and imaginary components of the impedance as a function of frequency, $\\omega$. Similarly, the real part of the impedance spectrum can be calculated from the imaginary part by\n",
+    "\n",
+    "$$\n",
+    "Z^{\\prime}(\\omega) = Z^{\\prime}(\\infty) + \\frac{2}{\\pi} \\int_0^\\infty{\\frac{xZ^{\\prime\\prime}(x) - \\omega Z^{\\prime\\prime}(\\omega)}{x^2 - \\omega^2}dx}\n",
+    "$$\n",
+    "\n",
+    "The residual error between the predicted and measured impedance can then be used to determine consistency with the Kramers-Kronig relations.\n",
+    "\n",
+    "Practically, however, the 0 to $\\infty$ frequency range required for integration can be difficult to measure experimentally, so several other methods have been developed to ensure Kramers-Kronig relations are met:\n",
+    "- [Measurement models](#Measurement-models)\n",
+    "- [The Lin-KK method](#The-Lin-KK-method)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Measurement models"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "import sys\n",
+    "sys.path.append(\"/Users/matt/Dropbox/UW-Files/Research/impedance.py/\")\n",
```

Comment:
Hard-coded path to sys for loading in the package

suggested change: sys.path.append("../../../")

Reply:
Good catch 👍

Reply:
There's another instance further down